### ISSUE TITLE
Address does not parse correctly

### ISSUE THREAD

**Issue by soapergem** [GreenBuildingRegistry/usaddress-scourgify#38]
Here's an example of a valid US address which is not parsed correctly by scourgify:

```
import json
import scourgify

components = scourgify.normalize_address_record("1509 Via Christina, Vista, CA 92084")
print(json.dumps(components, indent=2))
```

What happens here is that scourgify tries to change the City name to the street direction, and you end up with this:

```
{
  "address_line_1": "1509 VIA CHRISTINA VIS",
  "address_line_2": null,
  "city": null,
  "state": "CA",
  "postal_code": "92084"
}
```

I would obviously expect it to handle addresses like this correctly.

**Comment by soapergem** [comment]
I just created [a sister issue in the usaddress package](https://github.com/datamade/usaddress/issues/366) but even if they don't fix that, I think scourgify can still be more intelligent about things. If there is no PlaceName then perhaps just interpret the StreetNamePostType as the city?

**Comment by aavilla-riparian** [comment]
Just for documenting what is being seen:
```
pip list | grep usadd
usaddress           0.5.15
usaddress-scourgify 0.6.0
```
I get:
```
>>> usaddress.parse('1509 Via Christina, Vista, CA 92084')
[('1509', 'AddressNumber'), ('Via', 'StreetNamePreType'), ('Christina,', 'StreetName'), ('Vista,', 'StreetNamePostType'), ('CA', 'StateName'), ('92084', 'ZipCode')]
>>> usaddress.tag('1509 Via Christina, Vista, CA 92084')
({'AddressNumber': '1509', 'StreetNamePreType': 'Via', 'StreetName': 'Christina', 'StreetNamePostType': 'Vista', 'StateName': 'CA', 'ZipCode': '92084'}, 'Street Address')
```
----
Does your text come in tokenized already or is it just free form text? If tokenized then you can potentially use:

```python
from scourgify.normalize import normalize_addr_str

normalize_addr_str('1509 Via Christina', city='Vista', state='CA', zipcode='92084')
```
returns
```python
OrderedDict([('address_line_1', '1509 VIA CHRISTINA'), ('address_line_2', None), ('city', 'VISTA'), ('state', 'CA'), ('postal_code', '92084')])
```

**Comment by fablet** [comment]
> I just created [a sister issue in the usaddress package](https://github.com/datamade/usaddress/issues/366) but even if they don't fix that, I think scourgify can still be more intelligent about things. If there is no PlaceName then perhaps just interpret the StreetNamePostType as the city?

@soapergem I do not believe using StreetNamePostType in the absence of PlaceName is an appropriate solution.  I believe making such a design decision has too much potential to create more problems than it would solve.

Tokenizing your problem addresses, instead of sending them as a one line string, as described by @aavilla-riparian above is the correct solution to the situation you are facing.
